src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const li = document.querySelectorAll('li');
+
+function sortList(list) {
+  const sortedList = Array.from(list).sort((a, b) => {
+    const salaryA = +a.getAttribute('data-salary').slice(1).replaceAll(',', '');
+    const salaryB = +b.getAttribute('data-salary').slice(1).replaceAll(',', '');
+
+    return salaryB - salaryA;
+  });
+
+  return sortedList;
+}
+
+function getEmployees(list) {
+  const employees = [];
+
+  list.forEach((item) => {
+    const employee = {
+      name: item.textContent.trim().split(' ')[0],
+      position: item.getAttribute('data-position'),
+      salary: item.getAttribute('data-salary'),
+      age: item.getAttribute('data-age'),
+    };
+
+    employees.push(employee);
+  });
+
+  return employees;
+}
+
+const ul = document.querySelector('ul');
+const sortedLi = sortList(li);
```

Comment:
The `sortList` function is called correctly here to sort the list items based on salary. However, ensure that the `li` elements have the necessary `data-salary` attributes for the sorting to work as expected.